packages/react/src/components/ImageToolbar/mantine/ImageToolbar.tsx

```diff
@@ -5,169 +5,100 @@ import {
   InlineContentSchema,
   StyleSchema,
 } from "@blocknote/core";
-import {
-  Button,
-  FileInput,
-  LoadingOverlay,
-  Tabs,
-  Text,
-  TextInput,
-} from "@mantine/core";
-import {
-  ChangeEvent,
-  KeyboardEvent,
-  useCallback,
-  useEffect,
-  useState,
-} from "react";
+import { LoadingOverlay, Tabs } from "@mantine/core";
+import { FC, useState } from "react";
 
 import { useBlockNoteEditor } from "../../../hooks/useBlockNoteEditor";
 import { Toolbar } from "../../mantine-shared/Toolbar/Toolbar";
 import { ImageToolbarProps } from "../ImageToolbarProps";
+import { UploadPanel } from "./DefaultTabPanels/UploadPanel";
+import { EmbedPanel } from "./DefaultTabPanels/EmbedPanel";
 
+/**
+ * By default, the ImageToolbar component will render with default tabs.
+ * However, you can override the tabs to render by passing the `tabs` prop. You
+ * can use the default tab panels in the `DefaultTabPanels` directory or make
+ * your own using the `ImageToolbarPanel` component.
+ */
 export const ImageToolbar = <
   I extends InlineContentSchema = DefaultInlineContentSchema,
   S extends StyleSchema = DefaultStyleSchema
 >(
-  props: ImageToolbarProps<I, S>
+  props: ImageToolbarProps<I, S> & {
+    tabs?: {
```

Comment:
I'm not sure if providing a `tabs` prop as API is useful / maybe too much to maintain? IF we don't make it configurable like that, the code to add a panel is fairly easy because you re-use the default panels, right?

Reply:
I wanted to make the API the same as other UI components, i.e. that you can pass `children` to replace the default content. But tabs are a little different since each tab panel you add needs to correspond to a tab name. So I'm not sure there's a better way of making tabs configurable.